bench/protobuf.cc

```diff
@@ -0,0 +1,143 @@
+#include <benchmark/benchmark.h>
+
+#include <onnx/onnx.pb.h>
+
+inline void createValueInfo4D(
+    onnx::ValueInfoProto& value_info,
+    const std::string& name,
+    int64_t n,
+    int64_t c,
+    int64_t h,
+    int64_t w) {
+  value_info.set_name(name);
+
+  onnx::TypeProto_Tensor* tensor_type =
+      value_info.mutable_type()->mutable_tensor_type();
+  tensor_type->set_elem_type(onnx::TensorProto_DataType_FLOAT);
+
+  onnx::TensorShapeProto* shape = tensor_type->mutable_shape();
+  shape->add_dim()->set_dim_value(n);
+  shape->add_dim()->set_dim_value(c);
+  shape->add_dim()->set_dim_value(h);
+  shape->add_dim()->set_dim_value(w);
+}
+
+inline void createValueInfo2D(
+    onnx::ValueInfoProto& value_info,
```

Comment:
Please change `onnx::` to `ONNX_NAMESPACE::`. Or use `using`.

Reply:
Fixed